skyportal/handlers/api/sysinfo.py

```diff
@@ -26,14 +28,50 @@ def get(self):
                         properties:
                           invitationsEnabled:
                             type: boolean
-                          description: |
-                            Boolean indicating whether new user invitation pipeline
-                            is enabled in current deployment.
+                            description: |
+                              Boolean indicating whether new user invitation pipeline
+                              is enabled in current deployment.
+                          gitlog:
+                              type: array
+                              description: Recent git commit lines
+                          cosmology:
+                              type: string
+                              description: Details of the cosmology used here
+                          cosmoref:
+                              type: string
+                              description: Reference for the cosmology used.
         """
+        default_lines = 25
```

Comment:
I'm opting not to make this configurable. I think it's ok to be opinionated here.